src/app/(pages)/mobility.tsx

```diff
@@ -61,60 +117,123 @@ export default function Mobility(): JSX.Element {
                 setLoadingState(LoadingState.ERROR)
             })
     }
-
     return (
         <ScrollView
             refreshControl={
-                loadingState !== LoadingState.LOADING ? (
+                loadingState !== LoadingState.LOADING &&
+                loadingState !== LoadingState.LOADED ? (
```

Comment:
Refreshing with loaded data no longer works. Alternatively, we should at least implement a timeHook (like in the web app) that updates automatically.